app/styles/app.scss

```diff
@@ -1 +1,162 @@
+
+html {
+  font-size: 16px;
+}
+
+body {
+  padding: 5vmax;
+  background: $light-gray;
+  font-family: 'Source Sans Pro';
+  line-height: 1.5;
+}
+
+h1 {
+  color: $med-gray;
+  font-family: 'Merriweather';
+  font-size: 2.5rem;
+  text-align: center;
+  text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
+}
+
+.entries {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.select-menu {
+  position: relative;
+  max-width: 300px;
+  margin: 0 auto 20px auto;
+  background: #fff;
+  border-radius: 5px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.1);
+  
+  &:after {
+    position: absolute;
+    top: 50%;
+    right: 15px;
+    content: '\25BE';
+    font-family: 'sans-serif';
+    pointer-events: none;
+    transform: translateY(-50%);
+  }
+}
+
+.select {
+  display: block;
+  position: relative;
+  width: 100%;
+  top: -1px;
+  margin: 0;
+  padding: 15px;
+  appearance: none;
+  background: transparent;
+  border: 0;
+  border-radius: 0;
+  color: $dark-gray;
+  cursor: pointer;
+  line-height: 1;
+  font-weight: bold;
+
+  &::-ms-expand {
+    display: none;
+  }
+}
+
+.entry {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  margin-bottom: 5px;
+  padding: 10px 15px 12px 15px;
+  background: #fff;
+  border-radius: 5px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.1);
+  color: $med-gray;
+  font-weight: bold;
+  
+  &:last-child {
+    border-bottom: 0;
+  }
+  
+  &--header {
+    margin-bottom: 0;
+    background: transparent;
+    box-shadow: none;
+    color: $dark-gray;
+  }
+}
+
+.entry__date {
+  flex-shrink: 0;
+  width: 60px;
+}
+
+.entry__progress {
+  flex-shrink: 1;
+  position: relative;
+  width: 100%;
+  height: 25px;
+  top: 1px;
+  background: $light-gray;
+  box-shadow: 0 1px zpx rgba(0, 0, 0, 0.05) inset;
+  border-radius: 5px;
+  color: #fff;
+  overflow: hidden;
+  
+  .entry--aqi-1 & { background: linear-gradient(tint($aqi-1, 75%), tint($aqi-1, 85%)); }
+  .entry--aqi-2 & { background: linear-gradient(tint($aqi-2, 75%), tint($aqi-2, 85%)); }
+  .entry--aqi-3 & { background: linear-gradient(tint($aqi-3, 75%), tint($aqi-3, 85%)); }
+  .entry--aqi-4 & { background: linear-gradient(tint($aqi-4, 75%), tint($aqi-4, 85%)); }
+  .entry--aqi-5 & { background: linear-gradient(tint($aqi-5, 75%), tint($aqi-5, 85%)); }
+  
+  .progress-bar {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    padding: 0 10px;
+    background: $med-gray;
+    text-shadow: 0 1px 3px rgba(0, 0, 0, 0.25);
+    white-space: nowrap;
+    
+    .entry--aqi-1 & { background: linear-gradient(tint($aqi-1, 30%), shade($aqi-1, 5%)); }
+    .entry--aqi-2 & { background: linear-gradient(tint($aqi-2, 30%), shade($aqi-2, 5%)); }
+    .entry--aqi-3 & { background: linear-gradient(tint($aqi-3, 30%), shade($aqi-3, 5%)); }
+    .entry--aqi-4 & { background: linear-gradient(tint($aqi-4, 30%), shade($aqi-4, 5%)); }
+    .entry--aqi-5 & { background: linear-gradient(tint($aqi-5, 30%), shade($aqi-5, 5%)); }
```

Comment:
These `entry--api-n` classes might be a good candidate for a mixin -- perhaps just one where you pass in the two tint percentages?